prow/github/hmac.go

```diff
@@ -22,16 +22,17 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"errors"
-	"github.com/sirupsen/logrus"
-	"sigs.k8s.io/yaml"
 	"strings"
 	"time"
+
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/yaml"
 )
 
-// hmacSecret contains a hmac token and its expiration time.
+// hmacSecret contains a hmac token and the time when it's created.
 type hmacSecret struct {
-	Value  string    `json:"value"`
-	Expiry time.Time `json:"expiry"`
```

Comment:
The last time you checked it for k8s and istio Prow, they were still using the secret in the old format, which is a single plain-text HMAC token, so I assume no one is using it now.
But I have added it to the deprecation notice, just in case.